DESCRIPTION

```diff
@@ -16,6 +16,7 @@ RoxygenNote: 7.2.1
 Imports: 
     dplyr,
     magrittr,
+    metacoder,
```

Comment:
this is a cool package - i was unaware of this but looked at the example on their docs site and very cool and useful

Reply:
ya it's the best general tax viz i've encountered, especially since it works fairly seamlessly even as complexity scales. it's a little annoying to work with -- it's not the easiest to add your own data and colors that you want to plot, but shrug.